frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -98,13 +107,72 @@ export type InfoControlProps = {
  */
 const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderActionClick }) => {
   const popUpContext = React.useContext(PropertyPopUpContext);
+  const leaflet = useLeaflet();
+  const propertyInfo = popUpContext.propertyInfo;
+  const location = useLocation();
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const { getParcel } = useApi();
+  const jumpToView = () =>
+    leaflet.map?.setView(
+      [propertyInfo?.latitude as number, propertyInfo?.longitude as number],
+      MAX_ZOOM,
+    );
+  const zoomToView = () =>
+    leaflet.map?.flyTo(
+      [propertyInfo?.latitude as number, propertyInfo?.longitude as number],
+      MAX_ZOOM,
+    );
 
   useEffect(() => {
     const elem = L.DomUtil.get('infoContainer');
```

Comment:
What's this stuff for?

Reply:
unrelated to my changes, but this prevents scrolling on the slideout from zooming the map.